### ISSUE TITLE
1024 미팅

### ISSUE THREAD

**Issue by HyunjoonCho** [Team-KDA/natural-language-processing#2]
## 녹취록 요약
baseline은 baseline  
한계점이 많음   
* Cross Entropy; 특정 위치에 반드시 특정 단어가 와야 하는가? 요약 과정에서 순서가 바뀔 수도 있지 않나?
* 단순 transformer 모델을 실제로 사용하는 경우는 없을듯

## 가스-에너지 분야 문서 요약
한국 가스 공사 주관 - 10월 29일 신청 마감 done
일단은 3인 체제 유지..  
- [ ] BERT - 다 같이 공부 @HyunjoonCho 
- [ ] ROUGE 및 loss function @yukyeongLee 
- [ ] BERTSUM, KoBERT @lodikim
